Makefile

```diff
@@ -48,9 +52,12 @@ instrumentation/% :
 	printf "VERSION_TAG_PREFIX ?= $@/v\nGO_MODULE_NAME ?= github.com/instana/go-sensor/$@\n\ninclude ../../Makefile.release\n" > $@/Makefile
 	printf '// (c) Copyright IBM Corp. %s\n// (c) Copyright Instana Inc. %s\n\npackage %s\n\nconst Version = "0.0.0"\n' $(shell date +%Y) $(shell date +%Y) $(notdir $@) > $@/version.go
 
-fmtcheck:
-	@gofmt -l .
-	@test -z $(shell gofmt -l . && exit 1)
+fmtcheck: opisfmt $(MODULES)
```

Comment:
Is this bit still needed? 
It looks to me that we also need a rollback here..
The code from main branch is:

```ssh
fmtcheck:
	@gofmt -l .
	@test -z $(shell gofmt -l . && exit 1)
```

Could you please check?

Reply:
I have updated the makefile with changes to remove the check for excluded directories. Could you please review it ?